MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -15,162 +15,87 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
     
     @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
     
-    private var currentQuestionIndex = 0
-    private var correctAnswers = 0
-    private let questionsAmount: Int = 10
+    private var presenter: MovieQuizPresenter!
     
-    private var questionFactory: QuestionFactoryProtocol?
-    private var currentQuestion: QuizQuestion?
-    private var statisticService: StatisticService = StatisticServiceImplementation()
     private var alertPresenter: AlertPresenterProtocol?
     
     override func viewDidLoad() {
         super.viewDidLoad()
         imageView.layer.cornerRadius = 20
         imageView.layer.masksToBounds = true
-        
-        
-        questionFactory = QuestionFactory(moviesLoader: MoviesLoader(), delegate: self)
-        
-        
-        activityIndicator.startAnimating()
-        activityIndicator.hidesWhenStopped = true
-        
-        questionFactory?.loadData()
-        
-        statisticService = StatisticServiceImplementation()
-        
-        alertPresenter = ResultAlertPresenter(viewController: self)
+        presenter = MovieQuizPresenter(viewController:self)
+        showLoadingIndicator()
+        alertPresenter = AlertPresenter(viewController:self, presenter: presenter)
```

Comment:
Пройдись по написанному коду и поправь подобные стилевые ошибки
```suggestion
        alertPresenter = AlertPresenter(viewController: self, presenter: presenter)
```